frontend/packages/core/src/text.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import { Fab, Grid } from "@material-ui/core";
+
+import { ClipboardButton } from "./button";
+
+const CopyButtonContainer = styled(Grid)({
+  marginLeft: "7px",
+});
+
+const Pre = styled.pre({
+  border: "1px solid rgba(13, 16, 48, 0.38)",
+  backgroundColor: "rgba(13,16,48,0.12)",
+  borderRadius: "4px",
+  fontSize: "16px",
+  color: "#242b8c",
+  padding: "12px 16px",
+  flex: 1,
+  whiteSpace: "pre-wrap",
+  wordWrap: "break-word",
+  flexDirection: "row-reverse",
+  display: "flex",
+  overflowY: "scroll",
+});
+
+interface CodeProps {
+  children: string;
+  isCopiable?: boolean;
```

Comment:
nit: maybe just call this prop copyButton

copiable is a weird word and the prop also sounds like it might be doing clickjacking to prevent copying or something

Reply:
👍 , renamed to `showCopyButton` since `copyButton` sounds like it would expect a react element